src/pages/CreateInvoice.tsx

```diff
@@ -167,16 +209,34 @@ const CreateInvoice = () => {
   const { customer, organization, taxes } = data || {}
 
   const customerApplicableTax = useMemo(() => {
+    if (hasTaxProvider) return []
     if (!!customer?.taxes?.length) return customer?.taxes
-    return taxes?.collection
-  }, [customer, taxes])
 
-  // const localVatRate = customer?.vatRate || organization?.billingConfiguration?.vatRate || 0
+    return taxes?.collection
+  }, [customer?.taxes, hasTaxProvider, taxes?.collection])
 
   const [getAddOns, { data: addOnData }] = useGetAddonListForInfoiceLazyQuery({
     variables: { limit: 20 },
   })
 
+  const [getTaxFromTaxProvider] = useFetchDraftInvoiceTaxesMutation({
+    fetchPolicy: 'no-cache',
+    context: {
+      silentErrorCodes: [
+        LagoApiError.CurrencyCodeNotSupported,
+        LagoApiError.CustomerAddressCountryNotSupported,
+        LagoApiError.CustomerAddressCouldNotResolve,
+        LagoApiError.ProductExternalIdUnknown,
+        LagoApiError.AccountingTimeZoneNotSetForSeller,
+        LagoApiError.AccountingTimeZoneNotSupported,
+        LagoApiError.CustomerIdNotFound,
+        LagoApiError.ExternalServiceError,
+        LagoApiError.TaxDateTooFarInFuture,
+        LagoApiError.TaxDateTooFarInPast,
+      ],
```

Comment:
We could also create a variable for this array as it'll be needed too in other pages